pycode/memilio-epidata/memilio/epidata/getVaccinationData.py

```diff
@@ -1043,10 +966,9 @@ def get_vaccination_data(read_data=dd.defaultDict['read_data'],
     gd.write_dataframe(df_data_ageinf_state_cs,
                        directory, filename, file_format)
 
-    end_time = time.perf_counter()
+    timestamp_1 = time.perf_counter()
```

Comment:
Maybe there is a better name?

Reply:
I deleted all `time.perf_counter()` since I don't think it is still necessary.